toonz/sources/toonzfarm/tfarmserver/tfarmserver.cpp

```diff
@@ -49,6 +49,10 @@ using namespace std;
 #include <sys/sysctl.h> //To retrieve MAC HW infos
 #endif
 
+#ifdef LINUX
+#include <sys/sysctl.h> //For getting total memory
```

Comment:
According to sysconf(3), and on my glibc/linux system, `sysconf` is defined in `unistd.h` and not `sys/sysctl.h`.